docs/index.asciidoc

```diff
@@ -377,6 +381,19 @@ This parameter controls the keepalive time in seconds of the scrolling
 request and initiates the scrolling process. The timeout applies per
 round trip (i.e. between the previous scroll request, to the next).
 
+[id="plugins-{type}s-{plugin}-seearch_api"]
+===== `search_api`
+
+* Value can be any of: `auto`, `search_after`, `scroll`
+* Default value is `auto`
+
+`auto` uses `search_after` for Elasticsearch version `8.0.0` or higher, otherwise, fall back to `scroll` API.
```

Comment:
```suggestion
With `auto` the plugin uses the `search_after` parameter for Elasticsearch version `8.0.0` or higher, otherwise, the `scroll` API is used instead.
```